src/NServiceBus.Core.Tests/Transports/Learning/AsyncFileTests.cs

```diff
@@ -11,7 +11,7 @@ public class AsyncFileTests
         public async Task When_file_content_larger_than_buffer_size()
         {
             var originalContent = string.Join("", Enumerable.Repeat("a#~×ψؾࢯ‽%1", 2000));
-            const string filePath = "test.txt";
```

Comment:
this seems to be an issue when running via vstest. If we agree on this, we probably have to use the proposed solution on quite a few more tests